dfgviewer/modules/setup/index.php

```diff
@@ -115,28 +115,28 @@ protected function cmdAddMetadata() {
 		if (count($_ids) == $i) {
 
 			// Fine.
-			$_message = t3lib_div::makeInstance(
-				't3lib_FlashMessage',
+			$_message = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(
+				'\TYPO3\CMS\Core\Messaging\FlashMessage',
```

Comment:
Whenever `\TYPO3\CMS\Core\...` is used as a string, it has to be masked: `TYPO3\\CMS\\Core\\...`